CHANGELOG.md

```diff
@@ -64,6 +64,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [\#9457](https://github.com/cosmos/cosmos-sdk/pull/9457) Add amino support for x/authz and x/feegrant Msgs.
 * [\#9498](https://github.com/cosmos/cosmos-sdk/pull/9498) Added `Codec: codec.Codec` attribute to `client/Context` structure.
 * [\#9540](https://github.com/cosmos/cosmos-sdk/pull/9540) Add output flag for query txs command.
+* (x/slashing) [\#9458](https://github.com/cosmos/cosmos-sdk/pull/9458) Coins burned from slashing is now returned from Slash function and included in Slash event.
```

Comment:
this should go in a new "Features" subsection in the "Unreleased section"

(lol I feel like i've been typing this same sentence 10 times today in 10 different PRs, probably should have created empty subsections myself.)

Reply:
edit: Actually it should go in API breaking, right?

Reply:
ah right, missed that. [moved it to the unreleased feature section ](https://github.com/cosmos/cosmos-sdk/pull/9458/commits/424214feadb69ace82b3a2cf5215f5dbf426ca94)

Reply:
i think API breaking might make more sense, i can move it there